pkg/trace/version/version.go

```diff
@@ -26,6 +27,9 @@ func GetVersionDataFromContainerTags(containerID string, conf *config.AgentConfi
 	}
 	cTags, err := conf.ContainerTags(containerID)
 	if err != nil {
+		if err == config.ErrContainerTagsFuncNotDefined {
```

Comment:
Is it worth using something like `errors.Is` here instead of just doing equality checking?